src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread.h

```diff
@@ -120,13 +120,15 @@ class GenericThreadStackManagerImpl_OpenThread
     struct SrpClient
     {
         static constexpr uint8_t kServiceId           = 0x5d;
-        static constexpr uint8_t kMaxServicesNumber   = 3;
-        static constexpr uint8_t kMaxInstanceNameSize = 64;
-        static constexpr uint8_t kMaxNameSize         = 16;
+        static constexpr uint8_t kMaxServicesNumber   = CHIP_DEVICE_CONFIG_THREAD_SRP_MAX_SERVICES;
+        static constexpr uint8_t kMaxInstanceNameSize = chip::Mdns::kMdnsNameMaxSize;
+        static constexpr uint8_t kMaxNameSize         = chip::Mdns::kMdnsTypeMaxSize + chip::Mdns::kMdnsProtocolTextMaxSize + 1;
         static constexpr uint8_t kMaxHostNameSize     = 32;
-        static constexpr uint8_t kMaxTxtEntriesNumber = 4;
-        static constexpr uint8_t kMaxTxtValueSize     = 255;
-        static constexpr uint8_t kMaxTxtKeySize       = 16;
+        // Thread only supports operational discovery currently, increase when commissionable discovery would get added
```

Comment:
I would remove this comment. It is very likely that Thread devices will never support commissionable discovery, as it doesn't make much sense in their case, because Thread device cannot advertise over IP before being provisioned. By this comment you reminded me this existing issue, so I'm going to soon close it and remove TODO: comment.

Reply:
Correct - it would only make sense if a device, already on a Thread network by other means, would be able to join a CHIP network.
Will remove the `commissionable discovery` part